db/mongodb.go

```diff
@@ -223,3 +223,21 @@ func (m *MongoDB) GetCompany(cnpj string) (string, error) {
 	}
 	return string(b), nil
 }
+
+func (m *MongoDB) ExtraIndexes(idxs []string) error {
+	log.Output(1, "Creating the indexes...")
```

Comment:
Todo o repositório usa a elipses ao invês de três pontos nesses casos

 ```suggestion
	log.Output(1, "Creating the indexes…")
```